linfa-bayes/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "linfa-bayes"
+version = "0.1.0"
+authors = ["VasanthakumarV <vasanth260m12@gmail.com>"]
+description = "Collection of Naive Bayes Algorithms"
+edition = "2018"
+license = "MIT/Apache-2.0"
+repository = "https://github.com/rust-ml/linfa"
+readme = "README.md"
+keywords = ["factorization", "machine-learning", "linfa", "unsupervised"]
+categories = ["algorithms", "mathematics", "science"]
+
+[dependencies]
+ndarray = { version = "0.13" , features = ["blas"]}
+ndarray-linalg = { version = "0.12", features = ["openblas"] }
```

Comment:
you don't need the `ndarray-linalg` dependency, because you don't call any subroutine in your code

Reply:
Removed `ndarray-linalg`